rspace/README.md

```diff
@@ -24,7 +24,7 @@ Here are some key differences:
 
   * *produce*, which, given a piece data at a given channel, searches the store for matching continuation, using existing data in the store to help satisfy the match.
 
-For more information and detailed instructions on using `rspace`, see the tutorial [here](../../docs/rspace/Tutorial.md).
```

Comment:
We have the tutorial in 2 places now, the other would be:
https://github.com/rchain/rchain/blob/dev/rspace/src/main/tut/Tutorial.md
Shouldn't one be deleted to avoid confusion? @henrytill

Reply:
The tutorial in `src/main/tut` is the "source" which generates the tutorial found in `docs/rspace`.

Ideally we'd have a publishing mechanism which didn't require us to commit generated files, but this is a stopgap measure until that point in time.